csharp/ql/src/Security Features/CWE-327/InsecureSQLConnection.ql

```diff
@@ -48,6 +49,11 @@ class Sink extends DataFlow::Node {
 predicate isEncryptTrue(Source source, Sink sink) {
   sink.isEncryptedByDefault() and
   not source.setsEncryptFalse()
+  or
+  exists(ObjectCreation oc, Expr e | oc.getRuntimeArgument(0) = sink.asExpr() |
+    getInfoForInitializedConnEncryption(oc, e) and
+    e.getValue().toString().toLowerCase() = "true"
```

Comment:
```suggestion
    e.getValue().toLowerCase() = "true"
```
Does it work without the `toString`? Using `toString` in query logic is really bad practice since there are no stability requirements on the result of `toString` (i.e., `GitHub` could change the string tomorrow with no notice).

Reply:
It works, fixing it. BTW. Is there a better way to check the value for Booleans in C#? I have not find one as it seems to be a string value.

Reply:
I don't think there is anything nicer than what you've done, no. I think you have to settle with strings 😭